src/Pact/Repl.hs

```diff
@@ -242,6 +245,10 @@ pureEval ei e = do
   mode <- use rMode
   case r of
     Right a -> do
+        wref <- use (rEnv . eeWarnings)
+        warnings <- liftIO $ readIORef wref
+        traverse_ (outStrLn HOut . show . pretty) warnings
+        liftIO $ writeIORef wref []
```

Comment:
Sweet :+1:

Reply:
Isn't `show . pretty` defined as `showPretty` in our lib?

Reply:
I think it's `renderCompactStrict` or `renderPrettyString`. I can fix.